Flow.Launcher/App.xaml.cs

```diff
@@ -130,20 +131,17 @@ private void AutoStartup()
         //[Conditional("RELEASE")]
         private void AutoUpdates()
         {
-            Task.Run(async () =>
+            _ = Task.Run(async () =>
             {
                 if (_settings.AutoUpdates)
                 {
-                    // check udpate every 5 hours
-                    var timer = new Timer(1000 * 60 * 60 * 5);
-                    timer.Elapsed += async (s, e) =>
-                    {
-                        await _updater.UpdateAppAsync(API);
-                    };
-                    timer.Start();
-
-                    // check updates on startup
+                    // check update every 5 hours
+                    var timer = new PeriodicTimer(TimeSpan.FromHours(5));
                     await _updater.UpdateAppAsync(API);
+
+                    while (await timer.WaitForNextTickAsync())
+                        // check updates on startup
+                        await _updater.UpdateAppAsync(API);
```

Comment:
is this a double up of `await _updater.UpdateAppAsync(API);`? Doesnt it run once timer is instantiated?